src/coreclr/tools/aot/ILCompiler.Compiler/IL/ILImporter.Scanner.cs

```diff
@@ -952,10 +946,8 @@ private void ImportLdToken(int token)
                 }
                 _dependencies.Add(reference, "ldtoken");
             }
-            else if (obj is MethodDesc)
+            else if (obj is MethodDesc method)
```

Comment:
Nit: Apply the same style fix for `if (obj is TypeDesc)` above

Reply:
I didn't want to change this one either, but it was now a build break (warning).